build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    kotlin("jvm")
+    kotlin("plugin.allopen")
+
+    // The following line allows to load io.gatling.gradle plugin and directly apply it
+    id("io.gatling.gradle") version "3.9.0.2"
+    id("org.jlleitschuh.gradle.ktlint") version "11.0.0"
+    id("com.github.ben-manes.versions") version "0.44.0"
```

Comment:
We shouldn't add these plugins in our sample.
The goal is to demonstrate Gatling's gradle plugin, not give our opinion on dev practice and recommend some other plugins. We're assuming that people going with this sample are kotlin users and have their own opinion about the other plugins they want to use.
Please remove them.

Reply:
You're right. Will remove them